packages/flutter/lib/src/rendering/mouse_cursor.dart

```diff
@@ -416,45 +416,425 @@ class SystemMouseCursors {
+
+  /// A cursor indicating the status that the program is busy but can still be
+  /// interacted with.
+  ///
+  /// Typically the shape of an arrow with an hourglass or a watch at the corner.
+  /// Does *not* fall back to [wait] if unavailable.
+  ///
+  /// Corresponds to:
+  ///
+  ///  * Web: progress
+  ///
+  /// See also:
+  ///
+  ///  * [wait], which is similar to [progress] but the program can not be
+  ///    interacted with.
+  static const SystemMouseCursor progress = SystemMouseCursor._(kind: 'progress');
+
+  /// A cursor indicating somewhere the user can trigger a context menu.
+  ///
+  /// Typically the shape of an arrow with a small menu at the corner.
+  ///
+  /// Corresponds to:
+  ///
+  ///  * Android: TYPE_CONTEXT_MENU
+  ///  * Web: context-menu
+  ///  * macOS: contextualMenuCursor
+  static const SystemMouseCursor contextMenu = SystemMouseCursor._(kind: 'contextMenu');
+
+  /// A cursor indicating help information.
+  ///
+  /// Typically the shape of a question mark, or an arrow therewith.
+  ///
+  /// Corresponds to:
+  ///
+  ///  * Android: TYPE_HELP
+  ///  * Web: help
+  static const SystemMouseCursor help = SystemMouseCursor._(kind: 'help');
+
+
+  //// SELECTION ////
+
+  /// A cursor indicating selectable text.
   ///
   /// Typically the shape of a capital I.
+  ///
+  /// Corresponds to:
+  ///
+  ///  * Android: TYPE_TEXT
+  ///  * Web: text
+  ///  * macOS: IBeamCursor
   static const SystemMouseCursor text = SystemMouseCursor._(kind: 'text');
 
-  /// A cursor that indicates a forbidden action.
+  /// A cursor indicating selectable vertical text.
   ///
-  /// Typically the shape of a circle with a diagonal line.
-  static const SystemMouseCursor forbidden = SystemMouseCursor._(kind: 'forbidden');
+  /// Typically the shape of a capital I rotated to be horizontal. May fall back
+  /// to [text].
+  ///
+  /// Corresponds to:
+  ///
+  ///  * Android: TYPE_VERTICAL_TEXT
+  ///  * Web: vertical-text
+  ///  * macOS: IBeamCursorForVerticalLayout
+  static const SystemMouseCursor verticalText = SystemMouseCursor._(kind: 'verticalText');
+
+  /// A cursor indicating selectable table cells.
+  ///
+  /// Typically the shape of a hollow plus sign.
+  ///
+  /// Corresponds to:
+  ///
+  ///  * Android: TYPE_CELL
+  ///  * Web: cell
+  static const SystemMouseCursor cell = SystemMouseCursor._(kind: 'cell');
+
+  /// A cursor indicating precise selection, such as selecting a pixel in a
+  /// bitmap.
+  ///
+  /// Typically the shape of a crosshair.
+  ///
+  /// Corresponds to:
+  ///
+  ///  * Android: TYPE_CROSSHAIR
+  ///  * Web: crosshair
+  ///  * Web: crosshairCursor
+  static const SystemMouseCursor precise = SystemMouseCursor._(kind: 'precise');
+
 
-  /// A cursor that indicates something that can be dragged.
+  //// DRAG-AND-DROP ////
+
+  /// A cursor indicating moving something.
+  ///
+  /// Typically the shape of four-way arrow. May fall back to [allScroll].
```

Comment:
`allScroll` mentions falling back on `move`, and here it is vice versa. Is there a specific condition or precedence for one or the other?

Reply:
Yeah, I thought `move` should be ubiquitous so that `allScroll` might fall back to it, but it turns out no platforms have `move` but no `allScroll` at all. I'll remove this fallback.